packages/react-native/src/evm/providers/thirdweb-provider.tsx

```diff
@@ -74,6 +74,13 @@ export const ThirdwebProvider = <
     [supportedWallets],
   );
 
+  if (!thirdwebApiKey) {
+    console.warn(
+      "No API key provided. You will have limited access to thirdweb's services for storage, RPC, and account abstraction. You can get an API key from https://thirdweb.com/dashboard/",
```

Comment:
wait this will print on every re-render right? we should only print it once

Reply:
fixed